reporting-engine-worker/src/main/java/org/ihtsdo/termserver/scripting/reports/release/PackageComparisonReport.java

```diff
@@ -0,0 +1,199 @@
+	public static final String PREVIOUS_RELEASE = "Previous Release";
+
+	// Previous release zip archive name
+	public static final String PREVIOUS_RELEASE_PACKAGE = "Previous Release Package";
+
+	// Name of the current release - free text specified by the user
+	public static final String CURRENT_RELEASE = "Current Release";
+
+	// Current release zip archive name
+	public static final String CURRENT_RELEASE_PACKAGE = "Current Release Package";
+
+	// Published folder corresponding with thw release centre, e.g. ie, us, etc
+	public static final String PUBLISHED_FOLDER = "Published Folder";
+
+	// Name of the starting script
+	private static final String SCRIPT_NAME = "run_compare.sh";
+
+	private String previousRelease;
+	private String previousReleasePackage;
+	private String currentRelease;
+	private String currentReleasePackage;
+	private String publishedFolder;
+	
+	public static void main(String[] args) throws TermServerScriptException, IOException {
+		Map<String, String> params = new HashMap<>();
+
+		// Estonia release
+		params.put(PREVIOUS_RELEASE, "EE_2021-11-30");
+		params.put(PREVIOUS_RELEASE_PACKAGE, "SnomedCT_ManagedServiceEE_PRODUCTION_EE1000181_20211130T120000Z.zip");
+		params.put(CURRENT_RELEASE, "EE_2022-05-30");
+		params.put(CURRENT_RELEASE_PACKAGE, "SnomedCT_ManagedServiceEE_PRODUCTION_EE1000181_20220530T120000Z.zip");
+		params.put(PUBLISHED_FOLDER, "ee");
+
+		// Ireland release
+		/*
+		params.put(PREVIOUS_RELEASE, "IE_2022-04-21");
+		params.put(PREVIOUS_RELEASE_PACKAGE, "SnomedCT_ManagedServiceIE_PRODUCTION_IE1000220_20220421T120000Z.zip");
+		params.put(CURRENT_RELEASE, "IE_2022-10-21");
+		params.put(CURRENT_RELEASE_PACKAGE, "SnomedCT_ManagedServiceIE_PRODUCTION_IE1000220_20221021T120000Z.zip");
+		params.put(PUBLISHED_FOLDER, "ie");
+		*/
+
+		TermServerReport.run(PackageComparisonReport.class, args, params);
+	}
+
+	@Override
+	protected void preInit() throws TermServerScriptException {
+		// For running the report locally
+		runHeadless(5);
+	}
+
+	@Override
+	protected void init (JobRun run) throws TermServerScriptException {
+		ReportSheetManager.targetFolderId = "1od_0-SCbfRz0MY-AYj_C0nEWcsKrg0XA"; // Release Stats
+
+		previousRelease = run.getParamValue(PREVIOUS_RELEASE);
+		previousReleasePackage = run.getParamValue(PREVIOUS_RELEASE_PACKAGE);
+		currentRelease = run.getParamValue(CURRENT_RELEASE);
+		currentReleasePackage = run.getParamValue(CURRENT_RELEASE_PACKAGE);
+		publishedFolder = run.getParamValue(PUBLISHED_FOLDER);
+
+		super.init(run);
+	}
+
+	public void postInit() throws TermServerScriptException {
+		String[] columnHeadings = new String[] {
+				"Previous Release, Current Release, Comparison Results"
+		};
+		String[] tabNames = new String[] {
+				"Comparison Summary"
+		};
+		super.postInit(tabNames, columnHeadings, false);
+	}
+	
+	@Override
+	public Job getJob() {
+		JobParameters params = new JobParameters()
+				.add(PREVIOUS_RELEASE).withType(JobParameter.Type.STRING)
```

Comment:
I thought you were going to make these mandatory?   Also I mentioned the "withDescription()" method so you can say something about what you expect the user to insert here.

Reply:
@pgwilliams yes, sorry, missed this one :-(